docs/guides/serving.md

```diff
@@ -0,0 +1,68 @@
+Floyd run command has a [serve mode](../commands/run.md#serve). This will upload the files in the current directory and run a special command - `python app.py`. This file handles the incoming request, executes the logic and returns the output. Floyd expects this file to contain the code to run a web server(based on Flask).
+
+!!! IMPORTANT
+	Note that this feature is in *preview mode* and *is not production ready* yet.
+	Contact us at [support@floydhub.com](mailto:support@floydhub.com) for production ready serving support (in closed beta).
+
+!!! important
+	Before serving your model through REST API, you need to create a [floyd_requirements.txt](../commands/run.md##floyd_requirementstxt) and declare the flask requirement in it.
```

Comment:
This is a big gotcha and I've fallen for it a few times. Why can't we just do this ourselves? i.e auto add `flask` to the floyd_requirements.txt or install it in the base image. 

@houqp to comment

Reply:
yeah, we can inject this dependency from the backend.